### ISSUE TITLE
Exclude expired entries 

### ISSUE THREAD

**Issue by dennis-f** [MacPass/MacPassHTTP#38]
Right now, entries are passed to MacPassHTTP even if they're expired. The default behaviour should be not to show those entries as a possible result.

**Comment by mstarke** [comment]
I just looked into KeePassHTTP for reference and they seem to include expired entries as well.

SearchParameter by default do not exclude expired entires:

https://github.com/dlech/KeePass2.x/blob/4dba4a898d3cdcec7c0b66626237e37419df951a/KeePassLib/PwDefs.cs#L347

And they aren'e excluded in KeePasHTTP:

https://github.com/pfn/keepasshttp/blob/b9a6813bfc9ccc9618938249c1bfff744a28ca44/KeePassHttp/KeePassHttp.cs#L60

https://github.com/pfn/keepasshttp/blob/b9a6813bfc9ccc9618938249c1bfff744a28ca44/KeePassHttp/Handlers.cs#L117

**Comment by dennis-f** [comment]
You're right. I realized that too after returning to windows. The reason i did not close my issue was, that i think it should be implemented anyway. Although i should have edited the issue.

There is no reason why I should get presented entries that are marked as expired. That should be an issue in keepasshttp as well...

**Comment by dennis-f** [comment]
UPDATE: I found that option in KeePass > Tools > KeePassHttp Options... It's not in the browser extension. 

It was resolved in https://github.com/pfn/keepasshttp/pull/304

**Comment by mstarke** [comment]
I'll try to update the plugin to mirror the changes
